src/components/AppView/AppControls.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Redirect } from "react-router";
+
+import { IApp } from "../../shared/types";
+import ConfirmDialog from "../ConfirmDialog";
+
+interface IAppControlsProps {
+  app: IApp;
+  deleteApp: () => Promise<void>;
+}
+
+interface IAppControlsState {
+  modalIsOpen: boolean;
+  redirectToAppList: boolean;
+}
+
+class AppControls extends React.Component<IAppControlsProps, IAppControlsState> {
+  public state: IAppControlsState = {
+    modalIsOpen: false,
+    redirectToAppList: false,
+  };
+
+  public render() {
+    return (
+      <div className="AppControls">
+        <button className="button button-warning" disabled={true}>
```

Comment:
We don't have a `button-warning` class

Reply:
haha yes, I added it as a placeholder, but will remove!